### DISCUSSION TITLE
ccf-validation-tools/Knee/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#285] [General]
# ccf-validation-tools/Knee/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Knee/comments/

**Comment by emquardokus** [comment]
Knee_v1.2_DRAFT
https://docs.google.com/spreadsheets/d/1v3BfUp55Ty8Bl9ufHursLN9hXmyRcj0MHDbEpPwgFZc/edit#gid=1815525900

**Different labels**

FIXED    In row 21, the term CL:0000743 has different name/label in the source ontology. The name/label in the ASCT+B table is __ and the one in the ontology is hypertrophic chondrocyte. For reference, the given name/label by SMEs is Hypertrophic Chondrocyte. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

FIXED    In row 12, the term UBERON:0010996 has different name/label in the source ontology. The name/label in the ASCT+B table is Articular Cartilage of femur and the one in the ontology is articular cartilage of joint. For reference, the given name/label by SMEs is Articular cartilage of a joint. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

FIXED    In row 12, the term CL:1001607 has different name/label in the source ontology. The name/label in the ASCT+B table is articular chondrocyte of the femur and the one in the ontology is articular chondrocyte. For reference, the given name/label by SMEs is articular chondrocyte. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Comment by emquardokus** [comment]
Knee v1.3 https://docs.google.com/spreadsheets/d/1cQH2Pd2SGJcyUguaYgjqzXz7KwYnJmw1mZFd1WjbbgM/edit#gid=1815525900

**Different labels**
Switched columns of author preferred vs CL label to fix: The term CL:1001607 has a different name/label in the source ontology in the following 1 row 12. The name/label in the ASCT+B table is articular chondrocyte of the femur and the one in the ontology is articular chondrocyte. For reference, the given name/label by SMEs is articular chondrocyte. Please correct it in the columns AS/N/LABEL or CT/N/LABEL in the ASCT+B table.

**Additional updates**
Update Ref/1/DOI to Ref/1/ID to conform with new template
